cli/formatter/node.go

```diff
@@ -12,6 +12,8 @@ const (
 // getNodeType returns renderNode type.
 func getNodeType(node any) nodeType {
 	switch node.(type) {
+	case unorderedMap[any]:
```

Comment:
What about unorderedMap[string]?

Reply:
It's still `mapNodeType`. unorderedMap[string] satisfies unorderedMap[any] case.

Reply:
Anyway, because of the fix for comment below, unorderedMap[string] is no longer present int the code.